Mathlib/Algebra/Order/Hom/Basic.lean

```diff
@@ -95,7 +95,7 @@ theorem le_map_add_map_div [Group α] [AddCommSemigroup β] [LE β] [MulLEAddHom
     (a b : α) : f a ≤ f b + f (a / b) := by
   simpa only [add_comm, div_mul_cancel'] using map_mul_le_add f (a / b) b
 #align le_map_add_map_div le_map_add_map_div
-#align le_map_add_map_sub le_map_add_map_sub
+-- #align le_map_add_map_sub le_map_add_map_sub -- Porting note: TODO: `to_additive` clashes
```

Comment:
Oh, these are two declarations with the same additive version? Yeah, that happens sometimes.